doc/source/train/getting-started-pytorch-lightning.rst

```diff
@@ -190,6 +190,13 @@ Begin by wrapping your code in a :ref:`training function <train-overview-trainin
 
 Each distributed training worker executes this function.
 
+You can specify the input argument for `train_func` via the Trainer's `train_loop_config` parameter.
+
+.. warning::
+
+    Avoid passing large data objects through `train_loop_config` to reduce the
+    serialization and deserialization overhead. Instead, it's preferred to
+    initialize large objects (e.g. datasets, models) directly in `train_func`.
```

Comment:
Add a code snippet to show how to populate these? I think we want to show that it's a dictionary.

```python
def train_func(config):
    config[...]

config = {...}
trainer = TorchTrainer(train_func, train_loop_config=config, ...)
```

In the warning we can also show an example as well.

Reply:
Added two examples to
- highlight the config format
- show the good and bad practices of setting `train_loop_config`.